porting/debug-build/udev.rst

```diff
@@ -7,10 +7,12 @@ Now that you're logged in, you must create some udev rules to allow some tests t
 
```

Comment:
Please clarify that the above paragraph is for ON the device and please remove the . from codename.::